recipes/pypeit/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "pypeit" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pypeit/PypeIt/archive/{{ version }}.tar.gz
+  sha256: 9264c904e098c40c44e815d34d594305862a4c4fd0001c3ed81d24f706960ebf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - extension-helpers >=0.1
+  run:
+    - {{ cdt('mesa-libgl') }}  # [linux]
```

Comment:
This should probably be in build

Reply:
Actually, I'm not sure if it's necessary at all. I had added it to try and fix libgl.so.1 not being found during the test phase, but I think that that might actually be fixed by ``yum_requirements.txt``. I can try removing this, though.